modules/react-native-desktop-config/desktop/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+set(REACT_NATIVE_DESKTOP_EXTERNAL_MODULES_TYPE_NAMES ${REACT_NATIVE_DESKTOP_EXTERNAL_MODULES_TYPE_NAMES}
+  \"DesktopConfig\" PARENT_SCOPE)
+
+set(REACT_NATIVE_DESKTOP_EXTERNAL_MODULES_SRC ${REACT_NATIVE_DESKTOP_EXTERNAL_MODULES_SRC}
+                                              ${CMAKE_CURRENT_SOURCE_DIR}/desktopconfig.cpp PARENT_SCOPE)
+
+include_directories(../../../desktop)
+include(${CMAKE_ROOT}/Modules/ExternalProject.cmake)
```

Comment:
Should be safe to remove that `include` since CMake's [ExternalProject](https://cmake.org/cmake/help/v3.12/module/ExternalProject.html) module is not used in this file.

Reply:
Removed.